amundsen_application/static/js/components/ColumnList/index.tsx

```diff
@@ -29,6 +29,9 @@ import {
 
 import './styles.scss';
 
+export type SortDirection = 'asc' | 'desc';
```

Comment:
nit: I have a preference for the full words over abbreviations so that nobody misinterprets what things are meant to do. For example, here I have to use the context clue of `SortDirection` to understand that `desc` means `descending` as opposed to maybe `description`. So in favor of `ascending` and `descending`

Reply:
Oh, actually I was working on exactly that!
Using an enum, check the updated PR!